### DISCUSSION TITLE
can we still use extra addons after the update

### DISCUSSION THREAD

**Discussion by infinity-stack** [Deadlineem/Extras-Addon-for-YimMenu#138] [General]
can we or na im not tryna get banned

**Comment by Deadlineem** [comment]
> can we or na im not tryna get banned

Bans will come as they come, youre modding a game, with ANY menu.  You take that risk when you decide to use mods.  It still works as it should, but as always, use proper moderation when using mods and try to keep toxicity to a minimum so you dont get reported.

**Reply by infinity-stack** [reply]
yes i've been doing that for a while
